testing/dbc-examples/example.dbc

```diff
@@ -29,6 +29,17 @@ BO_ 1024 Amet: 8 Sit
 BO_ 1028 Dolor: 8 Sit
  SG_ OneFloat : 0|12@0+ (0.5,0) [0.00|130.00] "" Vector__XXX
 
+BO_ 200 SENSOR_SONARS: 8 SENSOR
```

Comment:
Is this from a known DBC file/device or did you make it up? I'd prefer to call it something that shows why it's here, like `MULTIPLEX_TEST` with `VARIANT_{A,B,…}_{1,2,…}` signals or something like that

Reply:
I took it from [here](https://github.com/technocreatives/dbc-codegen/issues/19#issuecomment-802653127) but my plan was on changing it before completion to avoid possible copyright issues.